### ISSUE TITLE
Custom components switch places

### ISSUE THREAD

**Issue by everyonesdesign** [springload/draftjs_exporter#55]
I have two custom components: image and direct speech. There's nothing very fancy in their code, e.g. image is rendered like this:

```py
class Image:
    def render(self, props):
        data = props['block']['data']
        return self.wrapper(data)

    def wrapper(self, data):
        caption = data.get('caption', '')

        return tags.div(
            {'class': 'contentImage'},
            self.image(data),
            DOM.create_text_node(caption))

    def image(self, data):
        src = data.get('image', '')
        caption = data.get('caption', '')
        return tags.img({
            'src': src,
            'class': 'contentImage-image',
            'alt': caption})
```

then I generate the content for the blocks (not important fields are dropped):

```
{  
   "blocks":[  
      {  
         "text":"Start text",
         "type":"unstyled",
         ...
      },
      {  
         "type":"image",
         "data":{  
            "caption":"Some image caption",
            "image":"http://placekitten.com/401/402"
         },
         ...
      },
      {  
         "text":"Middle text",
         "type":"unstyled",
         ...
      },
      {  
         "type":"direct-speech",
         "data":{  
            "name":"Steve Jobs",
            "image":"http://placekitten.com/500/501",
            "text":"Hello, world!!!!111"
         },
         ...
      },
      {  
         "text":"Last text",
         "type":"unstyled",
         ...
      }
   ]
   ...
}
```

I expect "Middle text" block to be between the image and direct speech, but for some reason these two blocks are rendered together and the text appears before them:

```html
<p>Start text</p>
<p>Middle text</p>
<div>
   <div class="contentImage">
      <img class="contentImage-image" src="http://placekitten.com/401/402">Some image caption
   </div>
   <blockquote class="directSpeech">
      <div class="directSpeech-image" style="background-image: url(http://placekitten.com/500/501);"></div>
      <div class="directSpeech-wrapper">
         <div class="directSpeech-title">
            Steve Jobs
         </div>
         <p class="directSpeech-text">Hello, world!!!!111</p>
      </div>
   </blockquote>
</div>
<p>Last text</p>
```

I hope the bug is reproducable; if you have any questions or need more details I'll provide you any information.

**Comment by thibaudcolas** [comment]
👍 Thanks for the report. I'll have a go at reproducing this and see what happens.

**Comment by everyonesdesign** [comment]
@thibaudcolas I noticed that the problem is reproduced when the components has the same wrapper tag value:
```
block_map = {**BLOCK_MAP, **{
    'direct-speech': {
        'element': DirectSpeech(),
        'wrapper': 'div',
    },
    'image': {
        'element': Image(),
        'wrapper': 'div',
    },
}}
```
If I change 'div' to 'None' here everything works perfect (but without the wrapper). Do I use the property in correct way?

**Comment by thibaudcolas** [comment]
This is now fixed via #57.

@everyonesdesign about your code, I have a couple of remarks / questions:

- Where does your `tags.div` method come from? I've been keen to introduce something like that in the exporter's API, but haven't had the chance yet.
- `DOM.create_text_node(caption)` – I don't think this is necessary, just passing `caption` should be enough.

**Comment by everyonesdesign** [comment]
@thibaudcolas 
`tags` a wrapper written not by me, but another developer in my team. The wrapper looks like:
```python
from functools import partial
from draftjs_exporter.dom import DOM


__all__ = ['tags']


class TagsFactory:
    """
    Eases creating of DOM elements for DraftJS exporter
    Use it like this:

    >>> tags = TagsFactory()
    >>> attributes = {}
    >>> content = 'some'
    >>> tags.a(attributes, content)
    """
    def __getattr__(self, item):
        return partial(DOM.create_element, item)


tags = TagsFactory()
```
